### PR TITLE
Cluedo - Nathalie 

### PR DESCRIPTION
**Pull request by nathalienordstrom**

### CODE REVIEW

File: README.md
Comment (nathalienordstrom) [inline README.md]: https://cluedo-nathalie.netlify.app

File: code/clue.js
Comment (ckjerrgren) [inline code/clue.js:191]: I think that if you had used the function that was suggested by the teachers 
```
  document.getElementById( 'killerName').innerHTML =`${mystery.killer.firstName}${mystery.killer.lastName}`
```
you could have gotten away with writing less code. Don't know if it was necessary to set the variables(objects?) as const first.

Comment (ckjerrgren) [reply code/clue.js:191]: Same with weapon and room :)

File: code/clue.js
Comment (ckjerrgren) [inline code/clue.js:226]: Nice use of if and else from last week

File: code/index.html
Comment (ckjerrgren) [inline code/index.html:9]: I like the favicon

File: code/style.css
Comment (ckjerrgren) [inline code/style.css:181]: Wow!

File: code/style.css
Comment (ckjerrgren) [inline code/style.css:112]: Some of the weapon pictures takes a bit longer to load than the rest.
Are some of them larger?

**Review by ckjerrgren — COMMENTED** [review]
Very nice website! 

I like all the little extra effects you've added and that it is responsive. 
Very readable code but you could have added your own comments to make it even easier for others to read and understand.

File: code/clue.js
Comment (EmOlsz) [inline code/clue.js]: Unnecessary blank line

File: code/clue.js
Comment (EmOlsz) [inline code/clue.js]: It's far more clean to initialise our `mystery` object already with properties that are going to be there at the end, but with initial value of `null`, like that :
```
const mystery = {
  killer: null,
  weapon: null,
  room: null,
}
```

File: code/clue.js
Comment (EmOlsz) [inline code/clue.js]: Wrong indentation everywhere in this function

File: code/clue.js
Comment (EmOlsz) [inline code/clue.js]: We don't need prefix `the` in every variable - let's delete that

File: code/style.css
Comment (EmOlsz) [inline code/style.css]: Let's wrap it up with quote marks, like that :
```
background-image: url("assets/cluedo_cover.jpeg");
```
Some browsers might not approve this value without quote marks

File: code/style.css
Comment (EmOlsz) [inline code/style.css]: We should avoid styling by ids, let's change that to class reference

File: code/style.css
Comment (EmOlsz) [inline code/style.css]: No need to add `px` when value is equal `0`. `margin-bottom: 0;` is completely ok

File: code/style.css
Comment (EmOlsz) [inline code/style.css]: unnecessary blank lines

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Good job Nathalie, congratulations on well done project :)
I would like you to check out both Claudia's and my comment and fix your code accordingly to them. After that, project will be approved!
Moreover, I would like you to fix logic of your application - right now it's possible to change suspect, weapon and room card value even after we reveal crime - it should be disabled

**Comment by nathalienordstrom** [comment]
a

**Comment by nathalienordstrom** [comment]
Have I not fixed all that you wanted or what's left?

**Comment by EmOlsz** [comment]
I might have missed some notifications about your changes, sorry.
Project is approved :)

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**
